src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/equalsavoidnull/InputEqualsAvoidNullTextBlocks.java

```diff
@@ -9,37 +9,37 @@
 package com.puppycrawl.tools.checkstyle.checks.coding.equalsavoidnull;
 
 public class InputEqualsAvoidNullTextBlocks {
-    public void equalsAvoid(String myString) {
-        if (myString.equals("stuff")) { // violation 'String literal expressions should be on the left side of an equals comparison.'
-        } // violation below 'String .* left .* of .* equals'
-        if (myString.equals("""
-                stuff""")) {
-        }
+  public void equalsAvoid(String myString) {
+    if (myString.equals("stuff")) { // violation 'String.*left.*of.*equals'
```

Comment:
Your string is 75 characters long.

Can you reuse 25 characters to keep message most meaningful part

Reply:
I've made a few changes, are they correct?

Reply:
@romani 
please verify these new changes